src/pages/Xvs/Table/index.tsx

```diff
@@ -1,9 +1,99 @@
 /** @jsxImportSource @emotion/react */
-import React from 'react';
-import { Paper } from '@mui/material';
+import React, { useContext, useMemo } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Typography } from '@mui/material';
+import { useUserMarketInfo } from 'clients/api';
+import { Token, Table, ITableProps } from 'components';
+import { AuthContext } from 'context/AuthContext';
+import { useTranslation } from 'translation';
+import { Asset, TokenId } from 'types';
+import { formatToReadablePercentage, formatCoinsToReadableValue } from 'utilities/common';
+import { useStyles } from '../styles';
 
-const TableUi: React.FC = () => <Paper />;
+interface IXvsTableProps {
+  assets: Asset[];
+}
 
-const Table: React.FC = () => <TableUi />;
+const XvsTableUi: React.FC<IXvsTableProps> = ({ assets }) => {
+  const history = useHistory();
+  const { t } = useTranslation();
+  const styles = useStyles();
 
-export default Table;
+  const columns = useMemo(
+    () => [
+      { key: 'asset', label: t('xvs.columns.asset'), orderable: false },
+      { key: 'xvsPerDay', label: t('xvs.columns.xvsPerDay'), orderable: true },
+      { key: 'supplyXvsApy', label: t('xvs.columns.supplyXvsApy'), orderable: true },
+      { key: 'borrowXvsApy', label: t('xvs.columns.borrowXvsApy'), orderable: true },
+    ],
+    [],
+  );
+
+  const rowOnClick = (e: React.MouseEvent<HTMLElement>, row: ITableProps['data'][number]) => {
+    history.push(`/market/${row[0].value}`);
+  };
+  // Format assets to rows
+  const rows: ITableProps['data'] = assets.map(asset => [
+    {
+      key: 'asset',
+      render: () => <Token symbol={asset.symbol as TokenId} />,
+      value: asset.id,
+    },
+    {
+      key: 'xvsPerDay',
+      render: () => (
+        <Typography variant="small1" css={[styles.whiteText, styles.fontWeight400]}>
```

Comment:
It doesn't look like `fontWeight400` is getting applied:
<img width="962" alt="Screen Shot 2022-05-17 at 15 48 07" src="https://user-images.githubusercontent.com/44675210/168840481-5617e537-4c51-43c6-9470-09fb30712a9d.png">

Seeing all the tables display their cell content with a font-weight of 400, I think we should update the table component directly so it applies globally

Reply:
I don't believe that you have the latest code because I see it applied. Applying it to the table would not be correct because we cannot predict what styles each column will require. The 600 font weight it is applied in the column render function as you can see so customizing those styles belongs here where they base style was applied.

Reply:
Yes you're right, I didn't have your last commit. Sorry for that.
Looks good to me!